app/src/main/res/values/styles.xml

```diff
@@ -37,4 +37,8 @@
         <item name="android:transitionName" tools:ignore="NewApi">toolbar</item>
     </style>
 
+    <style name="SplashScreen" parent="Theme.AppCompat.NoActionBar">
+        <item name="android:windowBackground">@drawable/activity_splash</item>
+        <item name="android:windowAnimationStyle">@null</item>
+    </style>
```

Comment:
```suggestion
    </style>

```

Reply:
I should probably add some stricter linting :sweat_smile: this codebase seems to wrap XML files with newlines.